src/CMakeLists.txt

```diff
@@ -14,3 +14,10 @@ add_subdirectory(vtrasterizer)
 if(CONTOUR_FRONTEND_GUI)
     add_subdirectory(contour)
 endif()
+
+if(CONTOUR_TESTING)
+    add_custom_target(test
+    COMMAND crispy_test
+    COMMAND vtbackend_test
+    COMMAND vtparser_test )
+endif()
```

Comment:
```suggestion
```

The cool thing is, we don't need any of it. For real :)

Let's just add the following into the project root's CMakeLists.txt:

```cmake
if(CONTOUR_TESTING)
    enable_testing()
endif()
```

and also of course keep CONTOUR_TESTING, but then we should be already ready to go.

CMake (CTest) automatically finds all test applications, because we added them deeper in the tree via `add_test` already.

Reply:
Updated